### ISSUE TITLE
Support timestamp option on prometheus import API

### ISSUE THREAD

**Issue by ledmonster** [VictoriaMetrics/VictoriaMetrics#750]
**Is your feature request related to a problem? Please describe.**

When I push metrics to `/api/v1/import/prometheus`, I want to specify a timestamp.

**Describe the solution you'd like**

```
/api/v1/import/prometheus?timestamp=1594370496905
```

**Describe alternatives you've considered**

add timestamps on prom file one by one

**Additional context**

our architecture:

```
IoT devices --> gateway --> pubsub --> worker --> VictoriaMetrics
```

IoT devices post prom formatted metrics (without timestamp) to gateway, and we push it to VictoriaMetrics through pubsub. Since there exist some latency on this path, I want to specify metrics timestamp by myself, instead of using current timestamp in VM import API layer.

**Comment by valyala** [comment]
VictoriaMetrics accepts `timestamp` as unix timestamp in milliseconds starting from the f95eea60d1a085b8fdaf8a9f8e1e0a6158ebb4a8 commit.

@ledmonster , could you build VictoriaMetrics from this commit and verify whether it works as expected? See [build instructions](https://victoriametrics.github.io/#how-to-build-from-sources). If everything works as expected, then the commit will be included in the next release.

**Comment by valyala** [comment]
The previous commit contained a bug, which could prevent from ingesting new data via `/api/v1/import/prometheus`. The bug has been fixed in the commit 62fde80490af6d4ab4acee7ce75e098334f37ae3 .

**Comment by ledmonster** [comment]
Thank you for quick implementation. I'll check the behavior hopefully on this weekend.

**Comment by valyala** [comment]
The implementation has been included in [v1.41.0](https://github.com/VictoriaMetrics/VictoriaMetrics/releases). @ledmonster , could you verify that it works as expected?

**Comment by ledmonster** [comment]
Thank you, it works as expected.

```
$ curl -v  --data-binary @node-exporter.prom  'http://localhost:8428/api/v1/import/prometheus?extra_label=job=curl&extra_label=instance=pc&timestamp=1599894041764'
```

```
$ curl -G 'http://localhost:8428/api/v1/query' --data-urlencode 'query=go_gc_duration_seconds[5m]' | jq . 
{
  "status": "success",
  "data": {
    "resultType": "matrix",
    "result": [
      {
        "metric": {
          "__name__": "go_gc_duration_seconds",
          "job": "curl",
          "instance": "pc",
          "quantile": "1"
        },
        "values": [
          [
            1599894041.764,
            "0.007358708"
          ]
        ]
      },
      {
        "metric": {
          "__name__": "go_gc_duration_seconds",
          "job": "curl",
          "instance": "pc",
          "quantile": "0.25"
        },
        "values": [
          [
            1599894041.764,
            "0.000048134"
          ]
        ]
      },
      {
        "metric": {
          "__name__": "go_gc_duration_seconds",
          "job": "curl",
          "instance": "pc",
          "quantile": "0.5"
        },
        "values": [
          [
            1599894041.764,
            "0.000071184"
          ]
        ]
      },
      {
        "metric": {
          "__name__": "go_gc_duration_seconds",
          "job": "curl",
          "instance": "pc",
          "quantile": "0"
        },
        "values": [
          [
            1599894041.764,
            "0.000009038"
          ]
        ]
      },
      {
        "metric": {
          "__name__": "go_gc_duration_seconds",
          "job": "curl",
          "instance": "pc",
          "quantile": "0.75"
        },
        "values": [
          [
            1599894041.764,
            "0.001780718"
          ]
        ]
      }
    ]
  }
}
```
